visualizations/frontend/other/TTable.vue

```diff
@@ -92,8 +92,10 @@
         "
         class="spanAllColumns center_cell"
       >
-        <div><i class="i-fa-solid i-fa-inbox"></i></div>
-        <div>{{ noDataMessage }}</div>
+        <slot name="no_data" >
+          <div><i class="i-fa-solid i-fa-inbox"></i></div>
```

Comment:
I don't see any icons in the design. Is this needed?

In addition, I think we might be moving away from fa icons and using more productcl ones, no?

Reply:
The icon is from the [pre-snyk table design](https://docs.google.com/spreadsheets/d/1854rrORMna_BKL7u73I140G9rtrivtcKVNi6EvMSXjs/edit#gid=0). 

Yeah I think we are moving away from fa icons, should probably be its own pitch to go through all the topcoat-public components and replace icons/remove fa dependency.

Reply:
I don't see the icons, do you?